.azure-pipelines/build-template.yml

```diff
@@ -21,6 +21,9 @@ parameters:
 - name: sonic_slave
   type: string
 
+- name: buildimage_artifact_name
```

Comment:
The value for this variable must be passed from "azure-pipelines.yml", I don't see that change in the PR.

Reply:
@smaheshm  Did you look at the pipeline details for this?  Doesn't even get to the point where the .azure-pipelines stuff is needed.  lgtm.yml is run first and fails... so this issue you have pointed out is irrelevant for now.